CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 ## Unreleased changes
 
 - Expose macros for deriving `Serial` and `Deserial` from `concordium-contracts-common`.
+- Address method `is_alias_of` is now `is_alias`.
+- Replaced `ReceiveName` and `InitName` with `OwnedReceiveName` and `OwnedContractName` from `concordium-contracts-common`.
+- Remove `ContractAddress` and `Address` in favor of their equivalents in `concordium-contracts-common`.
+- `AccountAddress::new` is replaced by a function called `account_address_from_registration_id`.
+- `Amount` now has a field `micro_ccd` instead of `microgtu`.
+- The default arithmetic (operator syntax, such as `+`, `-`, `*`) with `Amount` is now unchecked.
```

Comment:
What is the reasoning behind this change?

Reply:
The reason being that the implementation of these operators are already present in `concordium-contracts-common` but with the unchecked.
But, I also think, the default arithmetic should not be checked.